### ISSUE TITLE
Unable to write state or import aws_ses_configuration_set resource

### ISSUE THREAD

**Issue by dshmelev** [hashicorp/terraform-provider-aws#12939]
Hi,
The `aws_ses_configuration_set` resource doesn't import or write tf state.
 I try to create the resource but caught an error:
```
Error: Provider produced inconsistent result after apply

When applying changes to aws_ses_configuration_set.default, provider "aws"
produced an unexpected new value for was present, but now absent.

This is a bug in the provider, which should be reported in the provider's own
issue tracker.
```
I see terraform creates the resource, but It is absent in tf state. And it looks like a bug.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.24
+ provider.aws v2.58.0

### Affected Resource(s)

* aws_ses_configuration_set

### Terraform Configuration Files

```hcl
resource "aws_ses_configuration_set" "default" {
  name = "sesconfigset"
}
```

### Steps to Reproduce

1. The resource doesn't exist:
 ```
$ aws ses describe-configuration-set --configuration-set-name sesconfigset 
An error occurred (ConfigurationSetDoesNotExist) when calling the DescribeConfigurationSet operation: Configuration set <sesconfigset> does not exist.
```

2. Create `aws_ses_configuration_set` resource:
```
$ terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_ses_configuration_set.default will be created
  + resource "aws_ses_configuration_set" "default" {
      + id   = (known after apply)
      + name = "sesconfigset"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_ses_configuration_set.default: Creating...

Error: Provider produced inconsistent result after apply

When applying changes to aws_ses_configuration_set.default, provider
"registry.terraform.io/-/aws" produced an unexpected new value for was
present, but now absent.

This is a bug in the provider, which should be reported in the provider's own
issue tracker.
```

3. Check the state:
```
$ terraform state show aws_ses_configuration_set.default
No instance found for the given address!

This command requires that the address references one specific instance.
To view the available instances, use "terraform state list". Please modify
the address to reference a specific instance.
```

4. Make sure the resource is created:
```
$ aws ses describe-configuration-set --configuration-set-name sesconfigset 
{
    "ConfigurationSet": {
        "Name": "sesconfigset"
    }
}
```

5. Lets try to create resource again.
```
$ terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_ses_configuration_set.default will be created
  + resource "aws_ses_configuration_set" "default" {
      + id   = (known after apply)
      + name = "sesconfigset"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_ses_configuration_set.default: Creating...

Error: Error creating SES configuration set: ConfigurationSetAlreadyExists: Configuration set <sesconfigset> already exists.
        status code: 400, request id: 82dfdfd0-125b-452a-b386-1ea943f61e00

  on test.tf line 1, in resource "aws_ses_configuration_set" "default":
   1: resource "aws_ses_configuration_set" "default" {
```

6. Let's try to import the resource:
```
$ terraform import aws_ses_configuration_set.default sesconfigset
aws_ses_configuration_set.default: Importing from ID "sesconfigset"...
aws_ses_configuration_set.default: Import prepared!
  Prepared aws_ses_configuration_set for import
aws_ses_configuration_set.default: Refreshing state... [id=sesconfigset]

Error: Cannot import non-existent remote object

While attempting to import an existing object to
aws_ses_configuration_set.default, the provider detected that no object exists
with the given id. Only pre-existing objects can be imported; check that the
id is correct and that it is associated with the provider's configured region
or endpoint, or use "terraform apply" to create a new remote object for this
resource.
```

**Comment by anGie44** [comment]
Potentially fixed with #12024

**Comment by DrFaust92** [comment]
Closing for lack of response. Please reopen if still relevant.
